### PR TITLE
Social sharing

### PR DESCRIPTION
**Pull request by tianhuil**

### CODE REVIEW

**Comment by ludanin** [comment]
I've done some of the requested changes, I'm not 100% sure if I understood correctly what was to be done on `About`, [here's what I did](https://lu-mmb-mlx0j80gr.vercel.app/#/about).

Currently the toasts go away after 5 seconds, should we decrease this duration? Also, by default if an user is hovering over a toast this timer is paused, should we remove this feature?

The buttons a narrower now, but just let me know if you think they're still too wide, in addition the area of social buttons that contains the icon is now darker.

**Comment by tianhuil** [comment]
Thanks @ludanin: this isalmost there.
1. For the link sharing, I meant something more like this: https://mailmyballot.org/#/org/default/ where the Left Hand Side is an input field with the URL that selects when you click it.  It would also need the `https://` portion
2. For Facebook follow, is there a more specific anchor tag (it's just our page and I can't even figure out how to follow from there
3. For TW follow, is there a better message than "Now you can follow them"?  It's pretty bad

**Comment by ludanin** [comment]
I couldn't find anything about changing Twitter default pop-up message, neither [at the official docs](https://developer.twitter.com/en/docs/twitter-for-websites/follow-button/overview) nor [at publish.twitter.com](https://publish.twitter.com/?buttonHideUsername=on&buttonLarge=on&buttonType=FollowButton&lang=en&query=%40mailmyballot&widget=Button).

The facebook case is a bit tricky, there's a button that allows people to like the page, even from our website, but I've been facing some issues when styling this button. The necessary JavaScript needed to make the Like button work (available [here](https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v7.0) and [here](https://connect.facebook.net/en_US/sdk.js?hash=84478f802d19b65bc138d14cb464b919)) replaces any styling done by us. I don't think there'll be much to gain by trying to reverse these scripts. This is how the button will look like on our page:

Screenshot_20200613_171546

There's also a simpler version that is: 

Screenshot_20200613_172430

By the way, these are how the current buttons will look like when recolored:

Screenshot_20200613_171808

I actually enjoy them in this way, I would like to know if they would fit the new design when it rolls out. Should I push them to the repo?

Lastly, when searching for the Instagram colors, I've noticed we're using an older version of Font-Awesome. The Instagram icon has been updated and the most recent version uses the new icon.

**Comment by tianhuil** [comment]
@ludanin: I don't think you pushed your updates.  Could you do that?

RE FB / TW: let's keep the behavior as is.

I like the color of the follow buttons as well.  Consider applying that to the share buttons as well?

**Comment by ludanin** [comment]
I've pushed the changes now, also merged the latest changes to this branch.

[You can check the recolored buttons here](https://lu-mmb-bb65iudlk.vercel.app/#/about).

**Comment by michael-vote-at-home** [comment]
Thanks.  Could you do #1 from here: https://github.com/mail-my-ballot/mail-my-ballot/pull/28#issuecomment-643595826

File: packages/client/src/comp/SocialShare.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    flex: 1;
+    background-color: #0001;
+    box-shadow: inset -3px 0 4px #0004;
+    transition: background-color ease .5s;
+  }
+  & button:hover > i {
+    background-color: #0003;
+  }
+
+  /*
```
Comment (tianhuil) [inline packages/client/src/comp/SocialShare.tsx:52]: Thanks for documenting why this is necessary

**Pull request merged**